lib/Chat/SystemMessage/Listener.php

```diff
@@ -200,9 +214,21 @@ public static function register(IEventDispatcher $dispatcher): void {
 					continue;
 				}
 
-				$userJoinedFileRoom = $room->getObjectType() === 'file' &&
-						(!isset($participant['participantType']) || $participant['participantType'] !== Participant::USER_SELF_JOINED);
-				if ($userJoinedFileRoom || $userId !== $participant['actorId']) {
+				$participantType = null;
+				if (isset($participant['participantType'])) {
+					$participantType = $participant['participantType'];
+				}
+
+				$userJoinedFileRoom = $room->getObjectType() === 'file' && $participantType !== Participant::USER_SELF_JOINED;
+
+				// add a message "X joined the conversaion", whenever user $userId:
+				if (
+					// - has joined a file room but not through a public link
+					$userJoinedFileRoom
+					// - has been added by another user
+					|| $userId !== $participant['actorId']
```

Comment:
So the idea of this was always to not show that you added yourself, when you create a conversation. Maybe you can add this as a comment

Reply:
I checked and now when creating it also doesn't show up, maybe because when creating you join as "USER_MODERATOR"